bifrost.go

```diff
@@ -167,6 +167,8 @@ func newGoogleCloudStorage(bc *BridgeConfig) (RainbowBridge, error) {
 		EnableDebug:     bc.EnableDebug,
 		PublicRead:      bc.PublicRead,
 		UseAsync:        bc.UseAsync,
+		Object:          bc.Object,
+		Buckets:         bc.Buckets,
```

Comment:
there's absolutely no reason to update this struct for the purpose of your feature...this struct is for defining configuration parameters not function inputs.

Reply:
Got it. Thanks for the feedback.